arches_component_lab/src/arches_component_lab/widgets/ConceptMultiselectWidget/components/ConceptMultiselectWidgetEditor.vue

```diff
@@ -0,0 +1,123 @@
+<script setup lang="ts">
+import { ref, watchEffect } from "vue";
+import type { Ref } from "vue";
+
+import { useGettext } from "vue3-gettext";
+import TreeSelect from "primevue/treeselect";
+
+import { fetchConceptsTree } from "@/arches_component_lab/datatypes/concept/api.ts";
+import type { ConceptListValue } from "@/arches_component_lab/datatypes/concept/types.ts";
+
+import type { FormFieldResolverOptions } from "@primevue/forms";
+
+import type {
+    CollectionItem,
+    ConceptFetchResult,
+} from "@/arches_component_lab/datatypes/concept/types.ts";
+import GenericFormField from "@/arches_component_lab/generics/GenericFormField.vue";
+import { convertConceptOptionToFormValue } from "@/arches_component_lab/datatypes/concept/utils.ts";
+import type { AliasedNodeData } from "@/arches_component_lab/types.ts";
+
+const props = defineProps<{
+    graphSlug: string;
+    nodeAlias: string;
+    value?: ConceptListValue | string[] | null | undefined;
+}>();
+
+const selectedConcepts = ref<string[]>([]);
+if (
+    props.value &&
+    Object.hasOwnProperty.call(props.value, "node_value") &&
+    (props.value as ConceptListValue).node_value.length > 0
+) {
+    selectedConcepts.value = (props.value as ConceptListValue)
+        .node_value as string[];
```

Comment:
I don't think we need this if statement (assuming the above change).  
We should be able to set "selectedConcepts.value" directly.

```suggestion
    nodeAlias: string;
    value: ConceptListValue;
}>();

const selectedConcepts = ref<string[]>([]);
selectedConcepts.value = (props.value as ConceptListValue)
    .node_value as string[];
```